### ISSUE TITLE
C++ assertion error, invalid stock item ID

### ISSUE THREAD

**Issue by czack425** [Amulet-Team/Amulet-Map-Editor#80]
When opening a world using the 3D Editor, it will appear to load up to displaying "Setting up renderer" in the window.  The console shows the following:
```bash
19:52:33: Debug: Unrecognized accel key 'Import Construction', accel string ignored.
Traceback (most recent call last):
  File "/Users/SUR/Documents/git/Amulet-Map-Editor/amulet_map_editor/programs/__init__.py", line 132, in _page_change
    self._extensions[self.GetSelection()].enable()
  File "/Users/Jerry/Documents/git/Amulet-Map-Editor/amulet_map_editor/programs/edit/edit.py", line 48, in enable
    for arg in self._canvas.setup():
  File "/Users/Jerry/Documents/git/Amulet-Map-Editor/amulet_map_editor/programs/edit/canvas/edit_canvas.py", line 122, in setup
    self._tool_sizer = Tool(self)
  File "/Users/Jerry/Documents/git/Amulet-Map-Editor/amulet_map_editor/programs/edit/canvas/ui/tool/tool.py", line 44, in __init__
    self.register_tool("Import", SelectImportOperationUI)
  File "/Users/Jerry/Documents/git/Amulet-Map-Editor/amulet_map_editor/programs/edit/canvas/ui/tool/tool.py", line 61, in register_tool
    tool = tool_cls(self.canvas)
  File "/Users/Jerry/Documents/git/Amulet-Map-Editor/amulet_map_editor/programs/edit/canvas/ui/tool/tools/base_operation.py", line 38, in __init__
    self._operation_choice.SetItems(
  File "/Users/Jerry/Documents/git/Amulet-Map-Editor/amulet_map_editor/amulet_wx/ui/simple.py", line 107, in SetItems
    super().SetItems(self._keys)
  File "/Users/Jerry/python_envs/amulet/lib/python3.8/site-packages/wx/core.py", line 2548, in _ItemContainer_SetItems
    self.Set(items)
wx._core.wxAssertionError: C++ assertion ""Assert failure"" failed at /Users/robind/projects/bb2/dist-osx-py38/build/ext/wxWidgets/src/common/stockitem.cpp(213) in wxGetStockLabel(): invalid stock item ID
```

Note: The world is on a modded (spigot) server, but no mods have been added as of yet.

**Comment by Podshot** [comment]
What OS are you using? What version of Amulet are you using and are you running from source or from a compiled build?

**Comment by czack425** [comment]
Running from source (repo checked out of latest tag v0.6.11.0) using Mac OS Catalina (10.15).

**Comment by gentlegiantJGC** [comment]
From a google search this looks to be an issue with OSX. I think we have had this reported before and was also on OSX.

Could I get you to do some debugging for us so we can resolve it as we don't have access to OSX.

Add this line

```py
        print(self._keys)
```
just before line 107 in `"/Users/Jerry/Documents/git/Amulet-Map-Editor/amulet_map_editor/amulet_wx/ui/simple.py"`

so that it looks like this

```py
            self._keys = [str(v) for v in self._values]
        print(self._keys)
        super().SetItems(self._keys)
        if default is not None and default in self._values:
```

Then try running the program again. It will still crash but it should tell us what the keys were and might help us solve it.

Google seemed to suggest it was due to an empty sequence being set

**Comment by czack425** [comment]
After adding the print line, it is showing the following three lines of strings.
```python
['end', 'nether', 'overworld']
['Clone', 'Delete Chunks', 'Fill', 'Replace', 'Waterlog']
['\tImport Construction', 'Import Bedrock .mcstructure', 'Import Schematic (legacy)']
```

After some checking on my own, the tab in the first string of the last list seems to be causing the issue. Cleaning the strings seems to remove the bug, adding `self._keys = list(map(str.strip, self._keys))` in place of the print statement on line 107 of "/Users/Jerry/Documents/git/Amulet-Map-Editor/amulet_map_editor/amulet_wx/ui/simple.py" allows the world to render and show.

**Comment by gentlegiantJGC** [comment]
Okay that was an attempt to force the construction format to be the first and default value in the list. I assume that keeps the same ordering so it should solve this issue

**Comment by gentlegiantJGC** [comment]
This should be fixed with eb027ae0b92e2859664fa6b181b72feac54278be
